static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -0,0 +1,1033 @@
+                  Releases <IconOpen size="xs" />
+                </ExternalLink>{' '}
+                and artifact names. Follow all of the steps below to get a readable stack
+                trace:
+              </p>
+              <CheckList>
+                <EventHasReleaseNameChecklistItem facts={sourceResolutionResults} />
+                <ReleaseHasUploadedArtifactsChecklistItem
+                  facts={sourceResolutionResults}
+                />
+                <ReleaseSourceFileMatchingChecklistItem facts={sourceResolutionResults} />
+                <ReleaseSourceMapMatchingChecklistItem facts={sourceResolutionResults} />
+              </CheckList>
+              {releaseNameProgressPercent === 1 ? (
+                <ChecklistDoneNote />
+              ) : (
+                <VerifyAgainNote />
+              )}
+            </TabPanels.Item>
+            <TabPanels.Item key="fetching">
+              <CheckList>
+                <ScrapingSourceFileAvailableChecklistItem
+                  facts={sourceResolutionResults}
+                />
+                <ScrapingSourceMapAvailableChecklistItem
+                  facts={sourceResolutionResults}
+                />
+              </CheckList>
+              {scrapingProgressPercent === 1 ? (
+                <ChecklistDoneNote />
+              ) : (
+                <VerifyAgainNote />
+              )}
+            </TabPanels.Item>
+          </StyledTabPanels>
+        </Tabs>
+      </Body>
+      <Footer>
+        <Link
+          to=""
+          onClick={e => {
+            e.stopPropagation();
+            openModal(modalProps => (
+              <FeedbackModal
+                featureName="sourcemaps-debugger"
+                feedbackTypes={['This was helpful', 'This was not helpful']}
+                {...modalProps}
+              />
+            ));
+          }}
+        >
+          Was this helpful? <IconMegaphone size="xs" />
+        </Link>
+      </Footer>
+    </Fragment>
+  );
+}
+
+function CheckListItem({children, title, status}: PropsWithChildren<CheckListItemProps>) {
+  return (
+    <ListItemContainer>
+      <CheckMarkContainer>
+        {
+          {
+            none: <IconCircle size="md" color="gray200" />,
+            checked: <IconCheckmark size="md" color="green300" isCircled />,
+            alert: <IconWarning size="md" color="yellow300" />,
+            question: <IconQuestion size="md" color="gray300" />,
+          }[status]
+        }
+        <Line className="source-map-debugger-modal-checklist-line" />
+      </CheckMarkContainer>
+      <ListItemContentContainer>
+        <ListItemTitleWrapper>
+          <ListItemTitle status={status}>{title}</ListItemTitle>
+        </ListItemTitleWrapper>
+        {children}
+      </ListItemContentContainer>
+    </ListItemContainer>
+  );
+}
+
+function InstalledSdkChecklistItem({facts}: {facts: SourceResoultionResults}) {
```

Comment:
nit: Why name it differently here?
```suggestion
function InstalledSdkChecklistItem({sourceResolutionResults}: {sourceResolutionResults: SourceResoultionResults}) {
```

Reply:
Whoops "facts" was the working title I chose for this variable/type :D